### ISSUE TITLE
Sharp steep mountains

### ISSUE THREAD

**Issue by Silentor** [Silentor/InfluenceTerrainDemo#58]
Current IDW Shepard based mountains (volcano like) is not enough. We need classic granite sharp tall mountains (Hymalaia like). Delaunay triangles barycentric interpolation over existing cell mesh. Depends on #57

**Comment by Silentor** [comment]
Not now, use modified Shepard interpolator
